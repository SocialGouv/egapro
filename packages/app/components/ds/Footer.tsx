import React from "react"
// import NextLink from "next/link"
import { Box, Container, List, ListItem, Text, Link, Flex } from "@chakra-ui/react"

import Logo from "@/components/ds/Logo"
import TextLink from "@/components/ds/TextLink"
import packageConfig from "../../package.json"
import { ExternalLinkIcon } from "@chakra-ui/icons"

function Footer() {
  const version = process.env.REACT_APP_VERSION || packageConfig.version

  return (
    <Box
      id="footer"
      as="footer"
      role="contentinfo"
      py={6}
      sx={{
        borderTop: "1px solid #E3E4ED",
        "@media print": {
          display: "none",
        },
      }}
    >
      <Container
        maxW="container.xl"
        sx={{
          "@media (min-width: 1024px)": {
            display: "flex",
            flexDirection: "row",
          },
        }}
      >
        <Box
          sx={{
            "@media (min-width: 640px)": {
              display: "flex",
              flexDirection: "row",
            },
          }}
        >
          <Box pr={8}>
            <Link href="https://travail-emploi.gouv.fr/" isExternal pt={2} sx={{ display: "block" }}>
              <Logo />
            </Link>
          </Box>

          <List
            fontSize="sm"
            sx={{
              "@media (max-width: 639px)": {
                marginTop: "var(--chakra-space-4)",
              },
            }}
          >
            <ListItem>
              <Link href="https://travail-emploi.gouv.fr/IMG/xlsx/referents_egalite_professionnelle.xlsx" isExternal>
                <Flex justify="center" align="center">
                  Télécharger la liste des référents Egapro (XLSX, 22 Ko) <ExternalLinkIcon ml={2} />
                </Flex>
              </Link>
            </ListItem>
            {/* <ListItem>
              <NextLink href="/mentions-legales">
                <Link>Mentions légales</Link>
              </NextLink>
            </ListItem>
            <ListItem>
              <Link href={process.env.PUBLIC_URL + "/a11y/declaration-accessibilite-rgaa-4-1.pdf"}>
                Accessibilité&nbsp;: partiellement conforme
              </Link>
            </ListItem>
            <ListItem>
              <NextLink href="/cgu">
                <Link>Conditions générales d'utilisation</Link>
              </NextLink>
            </ListItem>
            <ListItem>
              <NextLink href="/politique-confidentialite">
                <Link>Politique de confidentialité</Link>
              </NextLink>
            </ListItem> */}
          </List>
        </Box>

        <Box
          sx={{
            "@media (max-width: 1023px)": {
              marginTop: "var(--chakra-space-4)",
            },
            "@media (min-width: 1024px)": {
              marginLeft: "auto",
            },
          }}
        >
          <Text fontSize="sm" as="i">
            Index Egapro a été développé par les équipes de la fabrique numérique des ministères sociaux
          </Text>
          <Text fontSize="sm" mt={1}>
            Pour nous aider à l'améliorer{" "}
            <TextLink
              to="https://voxusagers.numerique.gouv.fr/Demarches/2240?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=73366ddb13d498f4c77d01c2983bab48"
              isExternal
            >
              donnez-nous votre avis
            </TextLink>
            {", "}
            <TextLink to={`https://github.com/SocialGouv/egapro/tree/${version}`} isExternal>
              contribuez sur Github
            </TextLink>
          </Text>
        </Box>
      </Container>
    </Box>
  )
}

export default Footer
