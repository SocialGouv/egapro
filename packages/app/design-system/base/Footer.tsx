import React from "react"

import { FooterBody } from "./FooterBody"
import { FooterBodyItem } from "./FooterBodyItem"
import { FooterBottom } from "./FooterBottom"
import { FooterBottomItem } from "./FooterBottomItem"
import { FooterBottomLink } from "./FooterBottomLink"
import { FooterContentLink } from "./FooterContentLink"
import { Logo } from "./Logo"

export const Footer = () => {
  return (
    <footer className="fr-footer" role="contentinfo" id="footer">
      <div className="fr-container">
        <FooterBody
          logo={
            <a href="#">
              <Logo />
            </a>
          }
          description="Représentation Équilibrée a été développé par les équipes de la fabrique numérique des ministères sociaux."
          items={
            <>
              <FooterBodyItem>
                <FooterContentLink href="https://legifrance.gouv.fr" target="_blank" rel="noreferrer">
                  legifrance.gouv.fr
                </FooterContentLink>
              </FooterBodyItem>
              <FooterBodyItem>
                <FooterContentLink href="https://legifrance.gouv.fr" target="_blank" rel="noreferrer">
                  legifrance.gouv.fr
                </FooterContentLink>
              </FooterBodyItem>
            </>
          }
        />
        <FooterBottom>
          <>
            <FooterBottomItem>
              <FooterBottomLink href="https://legifrance.gouv.fr">Plan du site</FooterBottomLink>
            </FooterBottomItem>
            <FooterBottomItem>
              <FooterBottomLink href="https://legifrance.gouv.fr">Mentions légales</FooterBottomLink>
            </FooterBottomItem>
          </>
        </FooterBottom>
      </div>
    </footer>
  )
}
