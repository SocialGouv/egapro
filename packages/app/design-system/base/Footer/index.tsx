import React, { FunctionComponent } from "react"

export type FooterProps = {}

const Footer: FunctionComponent<FooterProps> = ({ children }) => {
  return (
    <footer className="fr-footer" role="contentinfo" id="footer">
      <div className="fr-container">{children}</div>
    </footer>
  )
}

export default Footer
