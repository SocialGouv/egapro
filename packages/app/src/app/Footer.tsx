import { config } from "@common/config";

export const Footer = () => (
  <footer className="fr-footer" role="contentinfo" id="footer">
    <div className="fr-footer__top">
      <div className="fr-container">
        <div className="fr-grid-row fr-grid-row--start fr-grid-row--gutters">
          <div className="fr-col-12 fr-col-sm-3 fr-col-md-2">
            <p className="fr-footer__top-cat">Liens utiles</p>
            <ul className="fr-footer__top-list">
              <li>
                <a className="fr-footer__top-link" href="mailto:index@travail.gouv.fr" target="_blank" rel="noreferrer">
                  Contact support technique : index@travail.gouv.fr
                </a>
              </li>
              <li>
                <a
                  className="fr-footer__top-link"
                  href={`${config.apiv2_url}/public/referents_egalite_professionnelle.xlsx`}
                  target="_blank"
                  title="Télécharger la liste des référents au format xlsx"
                >
                  Télécharger la liste des référents Egapro - Dreets
                </a>
              </li>
            </ul>
          </div>
          <div className="fr-col-12 fr-col-sm-3 fr-col-md-2">
            <p className="fr-footer__top-cat">&nbsp;</p>
            <ul className="fr-footer__top-list">
              <li>
                <a className="fr-footer__top-link" href="/aide-index" target="_blank">
                  Consulter l'aide concernant l'index
                </a>
              </li>
              <li>
                <a className="fr-footer__top-link" href="/aide-proconnect" target="_blank">
                  Consulter l'aide ProConnect (ex MonComptePro)
                </a>
              </li>
            </ul>
          </div>
          <div className="fr-col-12 fr-col-sm-3 fr-col-md-2">
            <p className="fr-footer__top-cat">&nbsp;</p>
            <ul className="fr-footer__top-list">
              <li>
                <a
                  className="fr-footer__top-link"
                  href="https://travail-emploi.gouv.fr/droit-du-travail/egalite-professionnelle-discrimination-et-harcelement/indexegapro"
                  target="_blank"
                  rel="noreferrer"
                >
                  Consulter le site du Ministère du Travail - Index
                </a>
              </li>
              <li>
                <a
                  className="fr-footer__top-link"
                  href="https://travail-emploi.gouv.fr/droit-du-travail/egalite-professionnelle-discrimination-et-harcelement/representation-equilibree-f-h-dans-les-postes-de-direction-des-grandes/"
                  target="_blank"
                  rel="noreferrer"
                >
                  Consulter le site du Ministère du Travail - Représentation équilibrée
                </a>
              </li>
            </ul>
          </div>
          <div className="fr-col-12 fr-col-sm-3 fr-col-md-2">
            <p className="fr-footer__top-cat">&nbsp;</p>
            <ul className="fr-footer__top-list">
              <li>
                <a
                  className="fr-footer__top-link"
                  href={`https://github.com/SocialGouv/egapro/commit/${config.githubSha}`}
                  target="_blank"
                  rel="noreferrer"
                >
                  Contribuer sur Github
                </a>
              </li>
              <li>
                <a className="fr-footer__top-link" href="/stats" target="_blank">
                  Statistiques
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div className="fr-container">
      <div className="fr-footer__body">
        <div className="fr-footer__brand fr-enlarge-link">
          <a href="/" title="Accueil - Egapro - Ministère du Travail, de l'Emploi et de l'Insertion">
            <p className="fr-logo">
              Ministère <br />
              du Travail,
              <br /> et de l'emploi
            </p>
          </a>
        </div>
        <div className="fr-footer__content">
          <p className="fr-footer__content-desc">
            Index Egapro et Représentation équilibrée sont développés et maintenus par les équipes de la fabrique
            numérique des ministères sociaux.
          </p>
          <ul className="fr-footer__content-list">
            <li className="fr-footer__content-item">
              <a
                className="fr-footer__content-link"
                href="https://info.gouv.fr"
                target="_blank"
                rel="noopener noreferrer"
              >
                info.gouv.fr
              </a>
            </li>
            <li className="fr-footer__content-item">
              <a
                className="fr-footer__content-link"
                href="https://service-public.fr"
                target="_blank"
                rel="noopener noreferrer"
              >
                service-public.fr
              </a>
            </li>
            <li className="fr-footer__content-item">
              <a
                className="fr-footer__content-link"
                href="https://legifrance.gouv.fr"
                target="_blank"
                rel="noopener noreferrer"
              >
                legifrance.gouv.fr
              </a>
            </li>
            <li className="fr-footer__content-item">
              <a
                className="fr-footer__content-link"
                href="https://data.gouv.fr"
                target="_blank"
                rel="noopener noreferrer"
              >
                data.gouv.fr
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div className="fr-footer__bottom">
        <ul className="fr-footer__bottom-list">
          <li className="fr-footer__bottom-item">
            <a className="fr-footer__bottom-link" href="/declaration-accessibilite">
              Accessibilité : partiellement conforme
            </a>
          </li>
          <li className="fr-footer__bottom-item">
            <a className="fr-footer__bottom-link" href="/mentions-legales">
              Mentions légales
            </a>
          </li>
          <li className="fr-footer__bottom-item">
            <a className="fr-footer__bottom-link" href="/cgu">
              CGU
            </a>
          </li>
          <li className="fr-footer__bottom-item">
            <a className="fr-footer__bottom-link" href="/politique-de-confidentialite-v2">
              Politique de confidentialité
            </a>
          </li>
          <li className="fr-footer__bottom-item">
            <button
              className="fr-footer__bottom-link fr-icon-theme-fill fr-link--icon-left"
              aria-controls="fr-theme-modal"
              data-fr-opened="false"
            >
              Paramètres d'affichage
            </button>
          </li>
        </ul>
        <div className="fr-footer__bottom-copy">
          <p>
            Sauf mention contraire, tous les contenus de ce site sont sous{" "}
            <a href="https://github.com/SocialGouv/egapro/blob/master/LICENSE" target="_blank" rel="noreferrer">
              licence Apache 2.0
            </a>
          </p>
        </div>
      </div>
    </div>
  </footer>
);
