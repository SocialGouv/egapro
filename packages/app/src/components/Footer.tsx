/** @jsx jsx */
import { css, jsx } from "@emotion/core";
import { Link } from "react-router-dom";

import globalStyles from "../utils/globalStyles";
import Logo from "./Logo";

import { useColumnsWidth, useLayoutType } from "./GridContext";

function Footer() {
  const width = useColumnsWidth(2);
  const layoutType = useLayoutType();
  const version = process.env.VERSION || require("../../package.json").version;

  return (
    <footer
      css={[
        styles.footer,
        layoutType === "desktop" &&
          css({ marginLeft: -(width + globalStyles.grid.gutterWidth) })
      ]}
    >
      <div
        css={[
          styles.footerLeft,
          layoutType === "desktop" && css({ width }),
          styles.footerLeftPrint
        ]}
      >
        <a
          href="https://travail-emploi.gouv.fr/"
          target="_blank"
          rel="noopener noreferrer"
          css={[
            styles.containerLogo,
            layoutType === "desktop" && styles.containerLogoDesktop
          ]}
        >
          <Logo />
        </a>
      </div>

      <div css={styles.footerLinks}>
        <a
          href="https://travail-emploi.gouv.fr/droit-du-travail/egalite-professionnelle-discrimination-et-harcelement/questions-reponses-sur-le-calcul-de-l-index-de-l-egalite"
          target="_blank"
          rel="noopener noreferrer"
          css={styles.link}
        >
          retrouvez le simulateur au format Excel
        </a>
        <br />
        <Link to="/mentions-legales" css={styles.link}>
          mentions légales
        </Link>
      </div>

      <div css={styles.footerInfo}>
        <em style={styles.info}>
          L’outil Index Egapro a été développé par les équipes de l’incubateur
          des ministères sociaux
        </em>
        <span style={styles.info}>
          Pour nous aider à l'améliorer
          <a
            href="https://voxusagers.numerique.gouv.fr/Demarches/2240?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=73366ddb13d498f4c77d01c2983bab48"
            target="_blank"
            rel="noopener noreferrer"
            css={styles.infoLink}
          >
            donnez-nous votre avis
          </a>
          <a
            href={`https://github.com/SocialGouv/egapro/tree/v${version}`}
            target="_blank"
            rel="noopener noreferrer"
            css={styles.infoLink}
          >
            contribuez sur Github
          </a>
        </span>
      </div>
    </footer>
  );
}

const styles = {
  footer: css({
    backgroundColor: "#FFF",
    height: 80,
    marginTop: 54,
    flexShrink: 0,
    display: "flex",
    flexDirection: "row",
    alignItems: "center",
    borderTop: "1px solid #EFECEF",
    "@media print": {
      display: "none"
    }
  }),
  footerLeft: css({
    display: "flex",
    flexDirection: "row",
    marginLeft: globalStyles.grid.gutterWidth,
    marginRight: globalStyles.grid.gutterWidth,
    "@media print": {
      marginLeft: 0
    }
  }),
  footerLeftPrint: css({
    "@media print": {
      width: "auto"
    }
  }),
  containerLogo: css({
    marginLeft: "auto",
    marginRight: 0,
    textDecoration: "none",
    color: "currentColor"
  }),
  containerLogoDesktop: css({
    marginRight: 25
  }),

  footerLinks: { display: "flex", flexDirection: "column" as "column" },
  link: {
    fontSize: 10,
    lineHeight: "11px",
    color: globalStyles.colors.default,
    textDecoration: "underline",
    margin: "2px 0"
  },

  footerInfo: {
    flexGrow: 1,
    flexShrink: 1,
    flexBasis: "0%",
    display: "flex",
    flexDirection: "column" as "column",
    textAlign: "right" as "right",
    marginRight: globalStyles.grid.gutterWidth
  },
  info: {
    fontSize: 12,
    lineHeight: "15px",
    marginBottom: 4
  },
  infoLink: {
    color: globalStyles.colors.default,
    textDecoration: "underline",
    marginLeft: 8
  }
};

export default Footer;
