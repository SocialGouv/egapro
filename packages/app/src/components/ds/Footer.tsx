import { ExternalLinkIcon } from "@chakra-ui/icons";
import { Box, Container, Flex, Link, List, ListItem, Text } from "@chakra-ui/react";
import { config } from "@common/config";

import { Logo } from "./Logo";
import { TextLink } from "./TextLink";

export interface FooterProps {
  consultationMode?: boolean;
}

export const Footer = ({ consultationMode }: FooterProps) => {
  return (
    <Box
      id="footer"
      as="footer"
      role="contentinfo"
      py={6}
      sx={{
        borderTop: "1px solid #E3E4ED",
        "@media print": {
          display: "none",
        },
      }}
    >
      <Container
        maxW="container.xl"
        sx={{
          "@media (min-width: 1024px)": {
            display: "flex",
            flexDirection: "row",
          },
        }}
      >
        <Box
          sx={{
            "@media (min-width: 640px)": {
              display: "flex",
              flexDirection: "row",
            },
          }}
        >
          <Box pr={8}>
            <Link href="https://travail-emploi.gouv.fr/" isExternal pt={2} sx={{ display: "block" }}>
              <Logo />
            </Link>
          </Box>

          <List
            fontSize="sm"
            sx={{
              "@media (max-width: 639px)": {
                marginTop: "var(--chakra-space-4)",
              },
            }}
          >
            {!consultationMode && (
              <ListItem>
                <Link href="https://travail-emploi.gouv.fr/IMG/xlsx/referents_egalite_professionnelle.xlsx" isExternal>
                  <Flex justify="center" align="center">
                    Télécharger la liste des référents Egapro (XLSX, 22 Ko) <ExternalLinkIcon ml={2} />
                  </Flex>
                </Link>
              </ListItem>
            )}
            {/* <ListItem>
              <NextLink href="/mentions-legales">
                <Link>Mentions légales</Link>
              </NextLink>
            </ListItem>
            <ListItem>
              <Link href={process.env.PUBLIC_URL + "/a11y/declaration-accessibilite-rgaa-4-1.pdf"}>
                Accessibilité&nbsp;: partiellement conforme
              </Link>
            </ListItem>
            <ListItem>
              <NextLink href="/cgu">
                <Link>Conditions générales d'utilisation</Link>
              </NextLink>
            </ListItem>
            <ListItem>
              <NextLink href="/politique-confidentialite">
                <Link>Politique de confidentialité</Link>
              </NextLink>
            </ListItem> */}
          </List>
        </Box>

        <Box
          sx={{
            "@media (max-width: 1023px)": {
              marginTop: "var(--chakra-space-4)",
            },
            "@media (min-width: 1024px)": {
              marginLeft: "auto",
            },
          }}
        >
          <Text fontSize="sm" as="i">
            Index Egapro et Représentation équilibrée ont été développés par les équipes de la fabrique numérique des
            ministères sociaux.
          </Text>
          {consultationMode ? (
            <Text fontSize="sm" mt={1}>
              Contact : <TextLink to={"mailto:index@travail.gouv.fr"}>index@travail.gouv.fr</TextLink>
            </Text>
          ) : (
            <Text fontSize="sm" mt={1}>
              Pour nous aider à l'améliorer{" "}
              <TextLink to={`https://github.com/SocialGouv/egapro/commit/${config.githubSha}`} isExternal>
                contribuez sur Github
              </TextLink>
            </Text>
          )}
        </Box>
      </Container>
    </Box>
  );
};
