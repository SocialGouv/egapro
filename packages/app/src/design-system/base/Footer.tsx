import type { PropsWithChildren } from "react";
import { forwardRef } from "react";
import { Container } from "../layout/Container";

export const Footer = ({ children }: PropsWithChildren) => (
  <footer className="fr-footer" role="contentinfo" id="footer">
    <Container>{children}</Container>
  </footer>
);

export type FooterBodyProps = {
  description: string;
  items?: React.ReactNode;
  logo: React.ReactNode;
};

export const FooterBody = ({ children }: PropsWithChildren) => <div className="fr-footer__body">{children}</div>;

export const FooterBodyBrand = ({ children }: PropsWithChildren) => {
  return <div className="fr-footer__brand fr-enlarge-link">{children}</div>;
};

export const FooterBodyContent = ({ children }: PropsWithChildren) => {
  return <div className="fr-footer__content">{children}</div>;
};

export const FooterBodyContentDescription = ({ children }: PropsWithChildren) => {
  return <p className="fr-footer__content-desc">{children}</p>;
};

export const FooterBodyContentItems = ({ children }: PropsWithChildren) => {
  return <ul className="fr-footer__content-list">{children}</ul>;
};

export const FooterBodyItem = ({ children }: PropsWithChildren) => {
  return <li className="fr-footer__content-item">{children}</li>;
};

export const FooterBottom = ({ children }: PropsWithChildren) => {
  return (
    <div className="fr-footer__bottom">
      <ul className="fr-footer__bottom-list">{children}</ul>
      <div className="fr-footer__bottom-copy">
        <p>
          Sauf mention contraire, tous les contenus de ce site sont sous{" "}
          <a href="https://github.com/SocialGouv/egapro/blob/master/LICENSE" target="_blank" rel="noreferrer">
            licence Apache 2.0
          </a>
        </p>
      </div>
    </div>
  );
};

export const FooterBottomItem = ({ children }: PropsWithChildren) => {
  return <li className="fr-footer__bottom-item">{children}</li>;
};

export const FooterBottomLink = forwardRef<HTMLAnchorElement, React.AnchorHTMLAttributes<HTMLAnchorElement>>(
  ({ children, ...rest }, ref) => {
    return (
      <a className="fr-footer__bottom-link" ref={ref} {...rest}>
        {children}
      </a>
    );
  },
);

FooterBottomLink.displayName = "FooterBottomLink";

export type FooterContentLinkProps = React.AnchorHTMLAttributes<HTMLAnchorElement>;

export const FooterContentLink = ({ children, ...rest }: FooterContentLinkProps) => {
  return (
    <a className="fr-footer__content-link" {...rest}>
      {children}
    </a>
  );
};
