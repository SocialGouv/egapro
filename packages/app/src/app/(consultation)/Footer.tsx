"use client";

import { consentModalButtonProps } from "@codegouvfr/react-dsfr/ConsentBanner";
import { Footer, type FooterProps } from "@codegouvfr/react-dsfr/Footer";

export const ConsultationFooter = (props: Omit<FooterProps, "cookiesManagementLinkProps">) => {
  return (
    <>
      <Footer
        {...props}
        cookiesManagementButtonProps={{
          ...consentModalButtonProps.nativeButtonProps,
        }}
      />
    </>
  );
};
