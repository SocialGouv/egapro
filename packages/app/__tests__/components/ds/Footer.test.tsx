import { render } from "@testing-library/react"
import Footer from "@/components/ds/Footer"

test("should match snapshot", () => {
  const view = render(<Footer />)
  expect(view).toMatchSnapshot()
})
