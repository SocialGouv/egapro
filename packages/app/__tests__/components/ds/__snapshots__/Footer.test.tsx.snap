// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <footer
        class="css-1dthsy1"
        id="footer"
        role="contentinfo"
      >
        <div
          class="chakra-container css-14xld82"
        >
          <div
            class="css-v3i7xg"
          >
            <div
              class="css-111wug5"
            >
              <a
                class="chakra-link css-1xbuty4"
                href="https://travail-emploi.gouv.fr/"
                rel="noopener"
                target="_blank"
              >
                <img
                  alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
                  class="chakra-image css-6t4orf"
                  src="/icons/marianne.svg"
                />
              </a>
            </div>
            <ul
              class="css-2xogyv"
              role="list"
            >
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                  href="https://travail-emploi.gouv.fr/IMG/xlsx/referents_egalite_professionnelle.xlsx"
                  rel="noopener"
                  target="_blank"
                >
                  <div
                    class="css-gmuwbf"
                  >
                    Télécharger la liste des référents Egapro (XLSX, 22 Ko) 
                    <svg
                      class="chakra-icon css-417j41"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <g
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-width="2"
                      >
                        <path
                          d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                        />
                        <path
                          d="M15 3h6v6"
                        />
                        <path
                          d="M10 14L21 3"
                        />
                      </g>
                    </svg>
                  </div>
                </a>
              </li>
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                >
                  Mentions légales
                </a>
              </li>
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                  href="/a11y/declaration-accessibilite-rgaa-4-1.pdf"
                >
                  Accessibilité : partiellement conforme
                </a>
              </li>
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                >
                  Conditions générales d'utilisation
                </a>
              </li>
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                >
                  Politique de confidentialité
                </a>
              </li>
            </ul>
          </div>
          <div
            class="css-1tqkm5s"
          >
            <i
              class="chakra-text css-g5hezf"
            >
              Index Egapro a été développé par les équipes de la fabrique numérique des ministères sociaux
            </i>
            <p
              class="chakra-text css-iesxu2"
            >
              Pour nous aider à l'améliorer
               
              <a
                class="chakra-link css-1qsits3"
                href="https://voxusagers.numerique.gouv.fr/Demarches/2240?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=73366ddb13d498f4c77d01c2983bab48"
                rel="noopener"
                target="_blank"
              >
                donnez-nous votre avis
              </a>
              , 
              <a
                class="chakra-link css-1qsits3"
                href="https://github.com/SocialGouv/egapro/tree/1.21.0"
                rel="noopener"
                target="_blank"
              >
                contribuez sur Github
              </a>
            </p>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="css-1dthsy1"
      id="footer"
      role="contentinfo"
    >
      <div
        class="chakra-container css-14xld82"
      >
        <div
          class="css-v3i7xg"
        >
          <div
            class="css-111wug5"
          >
            <a
              class="chakra-link css-1xbuty4"
              href="https://travail-emploi.gouv.fr/"
              rel="noopener"
              target="_blank"
            >
              <img
                alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
                class="chakra-image css-6t4orf"
                src="/icons/marianne.svg"
              />
            </a>
          </div>
          <ul
            class="css-2xogyv"
            role="list"
          >
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
                href="https://travail-emploi.gouv.fr/IMG/xlsx/referents_egalite_professionnelle.xlsx"
                rel="noopener"
                target="_blank"
              >
                <div
                  class="css-gmuwbf"
                >
                  Télécharger la liste des référents Egapro (XLSX, 22 Ko) 
                  <svg
                    class="chakra-icon css-417j41"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <g
                      fill="none"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-width="2"
                    >
                      <path
                        d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                      />
                      <path
                        d="M15 3h6v6"
                      />
                      <path
                        d="M10 14L21 3"
                      />
                    </g>
                  </svg>
                </div>
              </a>
            </li>
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
              >
                Mentions légales
              </a>
            </li>
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
                href="/a11y/declaration-accessibilite-rgaa-4-1.pdf"
              >
                Accessibilité : partiellement conforme
              </a>
            </li>
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
              >
                Conditions générales d'utilisation
              </a>
            </li>
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
              >
                Politique de confidentialité
              </a>
            </li>
          </ul>
        </div>
        <div
          class="css-1tqkm5s"
        >
          <i
            class="chakra-text css-g5hezf"
          >
            Index Egapro a été développé par les équipes de la fabrique numérique des ministères sociaux
          </i>
          <p
            class="chakra-text css-iesxu2"
          >
            Pour nous aider à l'améliorer
             
            <a
              class="chakra-link css-1qsits3"
              href="https://voxusagers.numerique.gouv.fr/Demarches/2240?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=73366ddb13d498f4c77d01c2983bab48"
              rel="noopener"
              target="_blank"
            >
              donnez-nous votre avis
            </a>
            , 
            <a
              class="chakra-link css-1qsits3"
              href="https://github.com/SocialGouv/egapro/tree/1.21.0"
              rel="noopener"
              target="_blank"
            >
              contribuez sur Github
            </a>
          </p>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
